apps/omg_watcher_rpc/test/omg_watcher_rpc/web/controllers/in_flight_exit_test.exs

```diff
@@ -103,6 +103,17 @@ defmodule OMG.WatcherRPC.Web.Controller.InFlightExitTest do
              } = WatcherHelper.no_success?("/in_flight_exit.get_data", %{"txbytes" => in_flight_txbytes})
     end
 
+    @tag fixtures: [:web_endpoint, :db_initialized, :bob]
+    test "behaves well if input is a spent deposit", %{bob: bob} do
```

Comment:
it still doesn't "behave well", at least not fully. How about "provides a report on unsupported start IFE case if input is a spent deposit"?